rosdep/python.yaml

```
@@ -6304,6 +6304,17 @@ python3-datadog-pip:
   ubuntu:
     pip:
       packages: [datadog]
+python3-dateutil:
+  arch: [python-dateutil]
+  debian: [python3-dateutil]
+  fedora: [python3-dateutil]
+  gentoo: [dev-python/python-dateutil]
+  nixos: [python310Packages.python-dateutil]
```

I'm still learning how Nix works, but I believe we should use `python3Packages` as the prefix here instead of the current Python 3 version, which changes from time to time. I know that the packaging dashboard lists the version-specific prefix, but I believe `python3Packages` is some sort of alias for the current version.

In any case, nearly all of the existing Nix rules for Python packages use this syntax.

```suggestion
  nixos: [python3Packages.python-dateutil]
```